### ISSUE TITLE
Inconsistent variable naming (`delay-in-ms` vs `delay`)

### ISSUE THREAD

**Issue by sernamar** [pmonks/spinner#23]
## Bug Report
While I was trying this library, I noticed an inconsistency in the naming of the delay variable between the code, documentation, and unit tests:
- In the code, the variable is named `delay-in-ms`.
- In the docstring and unit tests, it is referred to as `delay`.

### Steps to Reproduce:
Based on the unit test example:
```clojure
(require '[progress.indeterminate :as pi])

(pi/animate! :opts {:delay (* pi/default-delay-ms 10)}
             (Thread/sleep 5000))
```

### Expected Result:
5 character indicators, as `default-delay-ms` is `100`, and we `sleep` for `5000` ms.

### Actual Result:
50 character indicators.

### Correct example:
Using `:delay-in-ms` works as expected:
```clojure
(require '[progress.indeterminate :as pi])

(pi/animate! :opts {:delay-in-ms (* pi/default-delay-ms 10)}
             (Thread/sleep 5000))
```

### Additional Context:
In the source code:
- var name:
https://github.com/pmonks/spinner/blob/d2d5e886dcf378fdc153a0c88d39518dbabed91d/src/progress/indeterminate.clj#L113-L117
- doctrings: 
https://github.com/pmonks/spinner/blob/d2d5e886dcf378fdc153a0c88d39518dbabed91d/src/progress/indeterminate.clj#L164
https://github.com/pmonks/spinner/blob/d2d5e886dcf378fdc153a0c88d39518dbabed91d/src/progress/indeterminate.clj#L180

In unit tests:
https://github.com/pmonks/spinner/blob/d2d5e886dcf378fdc153a0c88d39518dbabed91d/test/spinner/core_test.clj#L48-L57

**Comment by pmonks** [comment]
Good catch - thank you!  I expect to be able to fix this and cut a new release sometime this coming week.

**Comment by pmonks** [comment]
Fixed in release 2.0.238.
